src/site/topics/intro/introduction.rst

```diff
@@ -223,7 +223,7 @@ Can we Write Code Now?
 
         * We don't need to make an instance of the class to use the method
 
-    * A good example of this in Java is the `Math class <https://docs.oracle.com/javase/8/docs/api/java/lang/Math.html>`_.
+    * A good example of this in Java is the `Math class <https://docs.oracle.com/javase/8/docs/api/java/lang/Math.html>`_
```

Comment:
maybe better to link to jdk17  https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/lang/Math.html

Reply:
good catch